actions/chats.js

```diff
@@ -74,3 +99,52 @@ export const putChat = (name) => {
             });
     };
 };
+
+export const putUserInChat = (cid, userId) => {
+    return async (dispatch) => {
+        const addUserId = true;
+        const headers = await addAuthHeader(
+            {
+                'Content-Type': 'application/json',
+                Accept: 'application/json'
+            },
+            addUserId
+        );
+        const destinationCid = await cid;
```

Comment:
you don't need to await anything here or the next line.. await is only used when calling a function with a promise.
in fact you could just remove destinationCid and destinationUserId and pass cid and userId to the fetch below